### PR TITLE
Update applicationInsights-js - v1.0.7

### PR DESCRIPTION
**Pull request by KamilSzostak**
Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [x] Run `tsc` without errors.
- [x] Run `npm run lint package-name` if a `tslint.json` is present.

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
[trackDependency](https://github.com/Microsoft/ApplicationInsights-JS/blob/master/JavaScript/JavaScriptSDK.Interfaces/IAppInsights.ts#L83)
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "../tslint.json" }`.

### CODE REVIEW

File: applicationinsights-js/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for ApplicationInsights-JS v0.23.2
+// Type definitions for ApplicationInsights-JS v1.0.7
```
Comment (DanielRosenwasser) [inline applicationinsights-js/index.d.ts]: Can you restrict this to the MAJOR.MINOR version?

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

**Comment by DanielRosenwasser** [comment]
Can you also add a `tslint.json` containing  `{ "extends": "../tslint.json" }` in the directory?

File: applicationinsights-js/index.d.ts [hunk 2]
```diff
@@ -717,16 +721,16 @@ declare module Microsoft.ApplicationInsights {
...
-        * @param  absoluteUrl Full url
-        * @param  pathName Leave this parameter blank
-        * @param  totalTime Total time it took for AJAX request to complete
-        * @param  success Whether AJAX request succeeded or failed
-        * @param  resultCode Result code returned from AJAX call
-        * @param  method  HTTP verb that was used (GET, POST)
-        */
-        trackAjax(id: string, absoluteUrl: string, pathName: string, totalTime: number, success: boolean, resultCode: number, method?: string): any;
+         * Log a dependency call
+         * @param id    unique id, this is used by the backend o correlate server requests. Use Util.newId() to generate a unique Id.
```
Comment (aozgaa) [inline applicationinsights-js/index.d.ts]: Can you align the descriptions in some way (either one space between param name and description or have all descriptions start at the same column)?

Comment (KamilSzostak) [inline applicationinsights-js/index.d.ts]: Fixed.

**Comment by zhengbli** [comment]
@DanielRosenwasser @aozgaa pinging

**Pull request merged**